<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Admin_add_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_Author_textBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_SourceTypeTxtbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_QuoteTxtbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quote</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_SearchBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_GenreTxtbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_TitleTxtbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_SourceTxtbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_Load">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QTableView" name="Admin_QuotesTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>751</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_Search_TextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>571</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_DeleteBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_HomeBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>iQoutes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
